flow/layers/image_filter_layer.cc

```diff
@@ -31,9 +31,6 @@ void ImageFilterLayer::Preroll(PrerollContext* context,
   if (!context->has_platform_view && context->raster_cache &&
       SkRect::Intersects(context->cull_rect, paint_bounds())) {
```

Comment:
How many of these if statements have identical tests? Could this be added as a boilerplate method either in the base `Layer` or in the `PrerollContext`?